srcpkgs/nodejs-lts-10/template

```diff
@@ -31,38 +31,42 @@ replaces="iojs>=0"
 conflicts="nodejs nodejs-lts"
 provides="nodejs-runtime-0_1"
 
-if [ "$XBPS_WORDSIZE" -ne "$XBPS_TARGET_WORDSIZE" ]; then
-	nocross="host and target must have the same pointer size"
+if [ ! "$XBPS_WORDSIZE" == "$XBPS_TARGET_WORDSIZE" ]; then
+	nocross="Host and target must have the same pointer size."
 fi
 
 case "$XBPS_MACHINE" in
-	ppc64*) ;;
-	mips*|ppc*) hostmakedepends+=" libatomic-devel" ;;
-	*) ;;
+	'mips*'|'ppc'|'ppc-musl') hostmakedepends+=" libatomic-devel";;
```

Comment:
no need to quote them

Reply:
again this change is pointless, it was fine as it was

Reply:
Not resolved, unnecessary quoting